tests/AjaxCallActionsWithDynamicConsent/Controllers/HomeController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using AjaxCallActionsWithDynamicConsent.Models;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Identity.Web;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace AjaxCallActionsWithDynamicConsent.Controllers
+{
+    public class HomeController : Controller
+    {
+        private readonly ILogger<HomeController> _logger;
+        private readonly ITokenAcquisition _tokenAcquisition;
+
+        public HomeController(ILogger<HomeController> logger, ITokenAcquisition tokenAcquisition)
+        {
+            _logger = logger;
+            _tokenAcquisition = tokenAcquisition;
+        }
+
+        public IActionResult Index()
+        {
+            return View();
+        }
+
+        [AuthorizeForScopes(Scopes = new[] { "https://ccbcc.sharepoint.com/AllSites.Read" })]
+        public async Task<IActionResult> AjaxAction()
```

Comment:
nit: if you want, you can put string.empty here or "user.read" as that is what we'll use to call graph. we can also update this later after it's merged.

Reply:
Thanks.